addons/product_expiry/tests/test_stock_lot.py

```diff
@@ -80,51 +80,54 @@ def test_00_stock_production_lot(self):
         self.env['stock.lot']._alert_date_exceeded()
```

Comment:
Missing task id in commit message (check other commits too)

"However it" -> "However if"